styles.css

```diff
@@ -0,0 +1,157 @@
+*{box-sizing: border-box;}
+body{margin: 0;}
+
+.comtainer{
+    padding: 4px;
+    border: 1px solid rgb(2, 0, 10);
+    font-family: sans-serif;
+}
+
```

Comment:
great uses for the reusable classes gap , great way to be more efficient with classes.